src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -302,7 +303,7 @@ export type StudyViewURLQuery = {
     sharedCustomData?: string;
 };
 
-export type XVsYChart = {
+export type XVsYScatterChart = {
```

Comment:
XvsYScatterChart?    the XV looks weird to me.  like a roman numeral.  these are serious matters.

Reply:
Superbowl XVIXXVsYScatterChart